app/src/main/java/com/sdp/movemeet/LoginActivity.java

```diff
@@ -20,9 +20,9 @@
 
 public class LoginActivity extends AppCompatActivity {
 
-    EditText mEmail, mPassword;
-    Button mLoginBtn;
-    TextView mCreateBtn;
+    EditText email, password;
```

Comment:
Good job renaming those attributes according to the convention :)

Reply:
Thank you!